tests/measurements/test_var.py

```diff
@@ -35,15 +62,18 @@ def circuit(x):
 
         x = 0.54
         res = circuit(x)
-        expected = np.sin(x) ** 2
+        expected = [np.sin(x) ** 2, np.sin(x) ** 2] if isinstance(shots, list) else np.sin(x) ** 2
 
-        assert np.allclose(res, expected, atol=tol, rtol=0)
+        assert np.allclose(res, expected, atol=0.05, rtol=0.05)
```

Comment:
Again, can we keep the original tolerance when computing analytically?

Reply:
Done!